next.config.js

```diff
@@ -0,0 +1,9 @@
+// This file is used in the tests to mock the next.config.js file!!!
+// It's not actually used by the library, it's just a mock file for testing purposes.
```

Comment:
Suggestion: Put this file in /tests if it doesn't need to live in the root

Reply:
yep, I tried that but quickly moved it to the root to power through.
I'll try with Christian's suggestion of using a env variable which we can manually set for running tests

Reply:
Cool. 👍